drake/systems/sensors/vtk_util.cc

```diff
@@ -0,0 +1,50 @@
+#include "drake/systems/sensors/vtk_util.h"
+
+#include <Eigen/Dense>
+
+#include <vtkCellData.h>
+#include <vtkNew.h>
+#include <vtkPlaneSource.h>
+#include <vtkSmartPointer.h>
+#include <vtkTransform.h>
+#include <vtkUnsignedCharArray.h>
+
+#include "drake/math/rotation_matrix.h"
+
+namespace drake {
+namespace systems {
+namespace sensors {
+namespace {
+const double kRadToDeg = 57.29577951308232;
+}  // namespace
+
+vtkSmartPointer<vtkPlaneSource> VtkUtil::CreateSquarePlane(double size) {
+  vtkSmartPointer<vtkPlaneSource> plane =
+      vtkSmartPointer<vtkPlaneSource>::New();
+  const double half_size = size * 0.5;
+  plane->SetOrigin(-half_size, -half_size, 0.);
+  plane->SetPoint1(-half_size, half_size, 0.);
+  plane->SetPoint2(half_size, -half_size, 0.);
+  plane->Update();
+
+  return plane;
+}
+
+vtkSmartPointer<vtkTransform> VtkUtil::ConvertToVtkTransform(
+    const Eigen::Isometry3d& transform) {
+  auto position = transform.translation();
+  auto axis_angle = drake::math::rotmat2axis(transform.linear());
```

Comment:
you might just want to set the matrix elements, as in https://github.com/RobotLocomotion/director/blob/master/src/app/ddDrakeModel.cpp#L130